build/cqs.tsv

```diff
@@ -171,41 +172,42 @@ org.eclipse.emf:org.eclipse.emf.common	2.12.0	compile
 org.eclipse.emf:org.eclipse.emf.ecore	2.12.0	compile
 org.eclipse.emf:org.eclipse.emf.ecore.xmi	2.12.0	compile
 org.eclipse.xsd:org.eclipse.xsd	2.12.0	compile
+org.ejml:ejml-core	0.34	compile
+org.ejml:ejml-ddense	0.34	compile
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18159

Reply:
@jahhulbert-ccri does this work for you?  Were you suggesting a different version of EJML?

Reply:
Let's merge this as is, and I've opened a second ticket to upgrade to 0.36: https://geomesa.atlassian.net/browse/GEOMESA-2469